main.js

```diff
@@ -0,0 +1,58 @@
+const studentsData = [
+    {
+        firstName: 'Василий',
+        lastName: 'Петров',
+        admissionYear: 2019,
+        courseName: 'Java',
+    },
+    {
+        firstName: 'Иван',
+        lastName: 'Иванов',
+        admissionYear: 2018,
+        courseName: 'JavaScript',
+    },
+    {
+        firstName: 'Александр',
+        lastName: 'Федоров',
+        admissionYear: 2017,
+        courseName: 'Python',
+    },
+    {
+        firstName: 'Николай',
+        lastName: 'Петров',
+        admissionYear: 2019,
+        courseName: 'Android',
+    }
+];
+
+class User {
+    constructor(firstName, lastName, fullName) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.fullName = firstName + ' ' + lastName;
+    }
+}
+
+class Student extends User {
+    constructor(firstName, lastName, fullName, admissionYear, courseName, course) {
```

Comment:
check with console.logwhat comes in these variables. Now objects go to firstName and the other variables are undefined. You need wrap ```firstName, lastName, fullName, admissionYear, courseName, course``` to object, like:
```constructor({firstName, lastName, fullName, admissionYear, courseName, course}) {```